pytorch_lightning/loops/fit_loop.py

```diff
@@ -197,11 +213,19 @@ def on_advance_start(self) -> None:
             window_length=self.trainer.accumulate_grad_batches
         )
 
+        # reset tracking
+        self.progress.train.reset_on_epoch()
```

Comment:
This should go to the loops `reset` function I think.

Reply:
Done !